### PR TITLE
[suggestion] about:support - improvements

### PR DESCRIPTION
**Pull request by janekptacijarabaci**
Added:
- OS
- Safe Mode
- Places Database (integrity)

See:
https://bugzilla.mozilla.org/show_bug.cgi?id=1171677
https://bugzilla.mozilla.org/show_bug.cgi?id=1260522  ([524757](https://bugzilla.mozilla.org/show_bug.cgi?id=524757))
https://bugzilla.mozilla.org/show_bug.cgi?id=522668

\+ "Restart normally…"

---

I've created the new build (x32, Windows) and tested.

### CODE REVIEW

File: toolkit/xre/nsAppRunner.cpp [hunk 1]
```diff
@@ -2969,6 +2969,12 @@ XREMain::XRE_mainInit(bool* aExitFlag)
     gSafeMode = true;
 #endif
 
+#ifdef MOZ_CRASHREPORTER
```
Comment (wolfbeast) [inline toolkit/xre/nsAppRunner.cpp]: Please don't reintroduce moz_crashreporter conditional blocks.

**Comment by Lootyhoof** [comment]
@wolfbeast Can I request a `UI Changes` tag for commits like this? It will help me keep track of what needs updating theme-wise.

**Comment by wolfbeast** [comment]
I didn't think this would influence theming?

**Comment by Lootyhoof** [comment]
`/toolkit/themes/*/global/aboutSupport.css` is being edited, which will need to be mirrored in themes.

**Comment by wolfbeast** [comment]
Right, right. So basically anything that touches toolkit or browser css or images should be labeled with it.

**Comment by wolfbeast** [comment]
I'll rename the label to "Theme changes" to better describe its purpose. Changes in UI layout don't necessarily change the default theme.

**Comment by janekptacijarabaci** [comment]
@wolfbeast
You add the label `UXP-task`, please:

"Restart normally…"
"Places Database - verify" -> "Copy text to clipboard"

**Pull request merged**